### ISSUE TITLE
[Merge] Progeria with Progeroid Syndrome

### ISSUE THREAD

**Issue by megankraus** [monarch-initiative/mondo#6227]
**Mondo term (ID and Label)**
Progeroid Syndrome MONDO:0020732

**Reason for deprecation**
Listed under progeroid syndrome in the ontology is a list of diseases associated with this group defined by diseases of premature aging and common clinical manifestations. Also listed under progeroid syndrome is a term called progeria with 3 children (3 unique diseases that are also diseases of premature aging). There is no definition of progeria in the ontology. Further, upon looking up progeria, Mayo Clinic https://www.mayoclinic.org/diseases-conditions/progeria/symptoms-causes/syc-20356038 defines this term as a disease of premature aging also known as Hutchinson-Gilford progeria syndrome, however, these two terms should not be identical because not all progeroid syndromes are Hutchinson-Gilford progeria syndrome. Additionally, I do not think the 3 diseases currently listed as children of progeria should not be grouped separately from the other disorders of premature aging, they should be listed under progeroid syndrome and progeria should be merged with progeroid syndrome. 

**Term to be merged with**
Progeria MONDO:0015333
**Your nano-attribution (ORCID)** 
0000-0003-4830-7530
![Screen Shot 2023-04-26 at 4 03 36 PM](https://user-images.githubusercontent.com/127896848/234713237-e891c8ca-3460-406c-a680-ca8f62d84051.png)

**Comment by kanems** [comment]
I agree that progeria vs. progeroid syndrome needs a clearer definition, but I do not think it is ontologically correct to merge these two. The "progeria" record reflect the OMIM Phenotypic series, which has only 3 children- HGPS, NGPS and WRS. 
My understanding of the origins of the term 'progeroid' is that it describes disorders that are like progeria but somehow distinct; granted it's been a few years since I really worked on this topic, so my familiarity with the field is not current.

Also noting that many of the children of "progeroid syndrome" from @megankraus 's screenshot are no longer showing in the latest release on OLS4 viewer (https://www.ebi.ac.uk/ols4/ontologies/mondo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FMONDO_0015333?lang=en). You may want to switch to the new ontology viewer. I just remembered this recent ticket, too: https://github.com/monarch-initiative/mondo/issues/1567 (I think this other ticket may also help clarify the distinction between progeria and progeroid)

**Comment by sabrinatoro** [comment]
Thank you @megankraus for creating this issue, and thank you @kanems for your comment. 

@nicolevasilevsky we should probably ask experts whether "progeria" is really a different disease. The current MESHxref should definitely be moved to Hutchinson-Gilford progeria syndrome.
If "progeria" is a separate disease, we might need to remove it from being a child of "progeroid syndrome" (there was no evidence for this parentage).

**Comment by megankraus** [comment]
I found a great review of progeroid syndromes here https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4958309/ . 
It says " All progeroid syndromes are characterized by similar clinical features ([Table 1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4958309/table/DMM024711TB1/)), but their underlying mechanisms can vary depending on the mutated gene and the pathway that is consequently altered. Below, we have classified progeroid syndromes into two general categories based on the molecular pathway involved. The first group includes those syndromes caused by alterations in components of the nuclear envelope, such as Hutchinson-Gilford progeria syndrome (HGPS), Néstor-Guillermo progeria syndrome (NGPS), atypical progeria syndromes (APSs), restrictive dermopathy (RD) and mandibuloacral dysplasia (MAD). The second group consists of progeroid syndromes induced by mutations in genes involved in DNA-repair pathways, such as Werner syndrome (WS), Bloom syndrome (BS), Rothmund-Thomson syndrome (RTS), Cockayne syndrome (CS), xeroderma pigmentosum (XP), trichothiodystrophy (TTD), Fanconi anaemia (FA), Seckel syndrome (SS), ataxia telangiectasia (AT), ataxia telangiectasia-like disorder (ATLD), cerebroretinal microangiopathy with calcifications and cysts (CRMCC), and Nijmegen breakage syndrome (NBN). A subcategory of this group comprises dyskeratosis congenita (DC) and Hoyeraal-Hreidarsson syndrome (HHS), linked to mutations in components of the telomerase complex (see [Box 1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4958309/#DMM024711B1) for a glossary of terms) that cause telomere attrition."

**Comment by nicolevasilevsky** [comment]
@ahamosh could you please review this ticket and let us know your thoughts? Thank you! :)

**Comment by ahamosh** [comment]
All,  I am crazy busy for the next week, so can't look or think deeply.  Not sure about the three things in the OMIM PS1766670, but I don't think of short telomere syndromes as the progeroid syndromes.  Can you remind me about this after May 8?  I did ask Joanna to take a look at it.  Sorry!

**Comment by nicolevasilevsky** [comment]
Hi @ahamosh - no worries! We can discuss it on a future call. Thank you and have a good week!

**Comment by sabrinatoro** [comment]
**From meeting with Ada on October 15th, 2024.** 

**"progeria",** MONDO:0020732
- OMIMPS:176670; Ada said that they (OMIM) will delete this phenotypic series as it is not complete. 
- All children of this term should become children of "progeroid syndrome"

**"progeroid syndrome" vs "premature aging syndrome"**
- progeroid syndrome = global premature aging (the patient looks like an old person)
- premature aging syndrome = could be part of a person (e.g. a tissue) that age prematurely, but the aging is not global)
- Note: one needs to be _aging_ prematurely to have a "premature aging syndrome". Therefore, if a patient is born "old", this should not be considered a "premature aging". 

**Review of terms in the "premature aging syndrome" branch**
- 'premature aging syndrome, Okamoto type' - MONDO:0011149
     - https://omim.org/entry/601811
     - old record, it is a "N=1" report. 
     - Consider removing
- 'Flynn-Aird syndrome' - MONDO:0007624
     - https://omim.org/entry/136300
     - This is also an old record (1965)
     - Consider removing
- 'Hallermann-Streiff syndrome' - MONDO:0009318
     - https://omim.org/entry/234100
     - this disease is not a premature aging syndrome
     - TO DO: exclude SubClass_Of
- 'Ogden syndrome' - MONDO:0010457
     - https://omim.org/entry/300855
     - the aging-related phenotype is congenital (at birth), therefore this is not a "premature aging"
     - TO DO: exclude SubClass_Of
